gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -906,6 +913,102 @@ def compute_an_by_group_membership(
     return ht
 
 
+def generate_v4_genomes_callstats(ht: hl.Table, an_ht: hl.Table) -> hl.Table:
+    """
+    Generate the call stats for the v4.0 genomes release.
+
+    Merge call stats from the v3.1 release, the v3.1 AN of new v4.0 variants,
+    samples with updated population labels, added samples, and removed samples.
+
+    Also, compute filtering allele frequencies ('faf'), 'grpmax', and 'gen_anc_faf_max'
+    on the merged call stats.
+
+    :param ht: Table returned by `join_release_ht_with_subsets`.
+    :param an_ht: Table with the allele number for new variants in the v4.0 release.
+    :return: Table with the updated call stats for the v4.0 genomes release.
+    """
+    logger.info("Updating AN Table HGDP pop labels and 'oth' -> 'remaining'...")
+    an_ht = update_pop_labels(an_ht, POP_MAP)
+
+    logger.info("Merging call stats from diff pop samples and added samples...")
+    global_array = ht.index_globals().global_array
+    farrays = [ht.ann_array[i].freq for i in range(3)]
+    fmeta = [global_array[i].freq_meta for i in range(4)]
+    count_arrays = [global_array[i].freq_meta_sample_count for i in range(4)]
```

Comment:
Are you using range(4) for fmeta and count_arrays here, just to avoid long names later? It's just a bit confusing at the beginning. Can we make it range(3) for everythin? Then just use [4] later?

Reply:
Feel free to make any changes to the code so it's easier to understand as long as it maintains the correct functionality

Reply:
I'll keep it as it is.